src/cript/api/utils/save_helper.py

```diff
@@ -37,10 +101,13 @@ def get_uuid_from_error_message(error_message: str) -> str:
     UUID
         the UUID the API had trouble with
     """
+    bad_uuid = None
     if error_message.startswith("Bad uuid: "):
         bad_uuid = error_message[len("Bad uuid: ") : -len(" provided")].strip()
     if error_message.strip().startswith("Duplicate uuid:"):
         bad_uuid = error_message[len(" Duplicate uuid:") : -len("provided")].strip()
+    if bad_uuid is None or len(bad_uuid) != 36:
```

Comment:
why 36?

Reply:
That is length of a uuid in string.
I can write that better though.

Reply:
oh I see,

I think something like this might be helpful, but the current way is also fine with some comments
```python

uuid_str_max_length = 36

if bad_uuid is None or len(bad_uuid) != uuid_str_max_length:
```

Reply:
I found a better solution: 
`len(bad_uuid) != len(uuid.uuid4()): # Ensure we found a full UUID describing string (here tested against a random new uuid length.)`

Test the length against the length of an actual UUID.

Reply:
that is better, that way if the package ever gets updated then our code will still work.

However, it is a bit weird the way that we are generating a new UUID to check that the length are correct. From what I can tell it seems that UUID will always have a length of 36 regardless, so maybe we don't ever have to create a new UUID to check its length against.

Reply:
I would recommend adding a comment for where we got 36 from, otherwise it'll be hard for anyone reading our code to know where we got that from. Either comment or a variable should work.

Reply:
36 doesn't appear in the current version of the code anymore.